src/optimal_retention.rs

```diff
@@ -36,7 +35,11 @@ impl Round for f32 {
 const R_MIN: f32 = 0.70;
 const R_MAX: f32 = 0.95;
 
-#[derive(Debug, Clone, PartialEq)]
+/// Function type for post scheduling operations that takes interval, maximum interval,
+/// current day index, due counts per day, and a random number generator,
+/// and returns a new interval.
+pub type PostSchedulingFn = dyn Fn(f32, f32, usize, Vec<usize>, &mut StdRng) -> f32 + Send + Sync;
```

Comment:
is the following method ok?

```rs
struct Wrap(dyn Fn(f32, f32, usize, Vec<usize>, &mut StdRng) -> f32 + Send + Sync)
```

something like this

then `impl PartialEq` just return true (

`impl Debug` return `"<function>"`

Reply:
create a new type, bypass the orphan rule.

Reply:
see ec0b264